includes/view/AngelTypes_view.php

```diff
@@ -129,6 +131,15 @@ function AngelType_edit_view(AngelType $angeltype, bool $supporter_mode)
             $supporter_mode
                 ? form_info(__('Hide at Registration'), $angeltype->hide_register ? __('Yes') : __('No'))
                 : form_checkbox('hide_register', __('Hide at Registration'), $angeltype->hide_register),
+            $supporter_mode
+                ? form_info(__('angeltypes.hide.on.shift.view'), $angeltype->hide_on_shift_view ? __('Yes') : __('No'))
```

Comment:
```suggestion
                ? form_info(__('angeltypes.hide_shift_view'), $angeltype->hide_on_shift_view ? __('Yes') : __('No'))
```

Reply:
or at least replace the dots with underscores ^^

Reply:
done